src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import url("https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap");
```

Comment:
I don't see in your code where you use "Manrope" font family, if you don't use it remove it.